packages/turbo-types/scripts/generate-schema.ts

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+/* eslint-disable @typescript-eslint/no-unsafe-assignment -- we're messing with TypeScript internals here */
+/* eslint-disable @typescript-eslint/no-unsafe-call -- we're messing with TypeScript internals here */
```

Comment:
I generally prefer line disables instead of file disables, but understandable if that would be untenable here.

Reply:
doesn't hurt! I moved them in 0fc9e7a66fd8dee5bee26212f9d76cc062d582c3